app/api/search/specs/search.spec.js

```diff
@@ -872,11 +901,11 @@ describe('search', () => {
     it('should filter by template', async () => {
       userFactory.mock(undefined);
       const { options } = await search.autocomplete('en', 'en');
-      expect(options.length).toBe(4);
+      expect(options.length).toBe(6);
```

Comment:
@RafaPolit why all tests have a new +2 length ? the number of entities on fixtures have not changed, i guess adding the titles is the cause ?

Reply:
Yes, exactly so, and the fact that I added the language to the title.